injectproxy/routes.go

```diff
@@ -206,8 +206,23 @@ func (r *routes) enforceLabel(h http.HandlerFunc) http.Handler {
 
 		// Remove the proxy label from the query parameters.
 		q := req.URL.Query()
-		q.Del(r.label)
+		if q.Get(r.label) != "" {
+			q.Del(r.label)
+		}
 		req.URL.RawQuery = q.Encode()
+		// Remove the proxy label from the form.
+		if req.Form.Get(r.label) != "" {
+			req.Form.Del(r.label)
```

Comment:
This doesn't actually do anything, right? This form is not re-encoded anywhere or used anywhere else, no?

Reply:
right. let me remove it.